core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -159,6 +160,43 @@ private[spark] object ClosureCleaner extends Logging {
     clean(closure, checkSerializable, cleanTransitively, Map.empty)
   }
 
+  /**
+   * Try to get a serialized Lambda from the closure.
+   *
+   * @param closure the closure to check.
+   */
+  private def getSerializedLambda(closure: AnyRef): Option[SerializedLambda] = {
+    if (scala.util.Properties.versionString.contains("2.11")) {
+      return None
+    }
+    val isClosureCandidate =
+      closure.getClass.isSynthetic &&
+        closure
+          .getClass
+          .getInterfaces.exists{x: Class[_] => x.getName.equals("scala.Serializable") }
+
+    if (isClosureCandidate) {
+      try {
+        val res = inspect(closure)
```

Comment:
Inline res? just little stuff that might streamline this a bit, it doesn't matter